### ISSUE TITLE
🐛 CORS erros on DRep profile

### ISSUE THREAD

**Issue by Ryun1** [IntersectMBO/govtool#2566]
### Area

Delegation Pillar

### Domain

gov.tools

### Context

 - Console shows many CORs errors for https://gov.tools/drep_directory/drep17c2cvvx6d2xxwwwea6d9w4g8jnzprnxs3ag8mawf45kvvswfhgj
 
```
drep17c2cvvx6d2xxwwwea6d9w4g8jnzprnxs3ag8mawf45kvvswfhgj:1 Access to fetch at 'https://cardaspians.io/JasonMatthys.jpeg' from origin 'https://gov.tools' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
index-dcQdfROJ.js:89 
        
Fetching the DRep image failed, reason:  TypeError: Failed to fetch

Access to fetch at 'https://cardaspians.io/JasonMatthys.jpeg' from origin 'https://gov.tools' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

index-dcQdfROJ.js:89 
 GET https://cardaspians.io/JasonMatthys.jpeg net::ERR_FAILED
```

![image](https://github.com/user-attachments/assets/28a73784-cb46-4d2a-8f6c-6a6894a1a276)

### Steps to reproduce

- Go to https://gov.tools/drep_directory/drep17c2cvvx6d2xxwwwea6d9w4g8jnzprnxs3ag8mawf45kvvswfhgj
- Open console

### Actual behavior

 - Loads of errors

### Expected behavior

- Errorless!

**Comment by bosko-m** [comment]
@Ryun1 Probably duplicate of #2546 which didnt yet end up on production.

**Comment by Ryun1** [comment]
i believe this is fixed now
